engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/function/FunctionSchema.java

```diff
@@ -264,4 +231,77 @@ public boolean isFullyApplied(InvokeCallableNode.DefaultsExecutionMode defaultsE
   public boolean isFullyApplied() {
     return isFullyApplied;
   }
+
+  public static final class Builder {
+    private boolean isProjectPrivate = false;
+    private Annotation[] annotations = new Annotation[] {};
```

Comment:
Why don't you use `new Annotation[0]`?

Reply:
I don't know. Is is better to use `new Annotation[0]` for the JVM? I thought this is just a coding style issue.

Reply:
Both forms are compiled to the same code. Yes, this is just a coding issue. Your style is very unusual, I'd say.